README.md

```diff
@@ -0,0 +1,254 @@
+![NGINX Banner](docs/img/logo.png "NGINX Banner")
+
+NGINX (pronounced "engine x" or "en-jin-eks") is the world's most popular Web Server, high performance Load Balancer, Reverse Proxy, API Gateway and Content Cache.
+
+NGINX is free and open source software, distributed under the terms of a simplified [2-clause BSD-like license](LICENSE.md).
+
+A commercial version, [NGINX Plus](https://www.f5.com/products/nginx/nginx-plus), with additional enterprise features and support, is available from [F5, Inc](https://www.f5.com/).
```

Comment:
```suggestion
Enterprise distributions, commercial support and training are available from [F5, Inc](https://www.f5.com/products/nginx).
```
This is the phrasing I'm working with for nginx.org - there is a lot more than just NGINX Plus now.

Reply:
Updated in the final version